src/components/challengeCard.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+import Text from "~/components/text";
+import { BsNut } from "react-icons/bs";
+
+import React from "react";
+import { useState } from "react";
+
+// Reusable Text Component for challenge descriptions
+
+// Main Challenge Card Component
+const ChallengeCard: React.FC<{
+  title: string;
+  type: string;
+  description: string;
+  points: number;
+  difficulty: string;
+  solves: number;
+}> = ({ title, type, description, points, difficulty, solves = 0 }) => {
+  const [isHovered, setIsHovered] = useState(false);
+  {
+    return (
+      <div
+        style={isHovered ? { opacity: 0.8 } : { opacity: 1 }}
+        onMouseEnter={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+      >
+        <div style={cardStyle}>
+          <div style={contentStyle}>
+            <BsNut size={52} style={{ transform: "rotate(90deg)" }} />
+            <Text
+              variant="secondary"
+              glow="secondary"
+              style={{ fontSize: "24px", fontWeight: "bold" }}
+            >
+              <br></br>
+              {title}
+            </Text>
+            <Text
+              variant="secondary"
+              glow="secondary"
+              style={{ fontSize: "16px" }}
+            >
+              {type}
+            </Text>
+            <Text
+              variant="secondary"
+              glow="secondary"
+              style={{ fontSize: "18px", fontWeight: "bold" }}
+            >
+              <br></br>
```

Comment:
Can you explain these `<br>` tags?

Reply:
It was a dirty fix to make better margins between the text lol